packages/web-components/fast-element/src/index.ts

```diff
@@ -43,4 +43,8 @@ export * from "./templating/node-observation.js";
 export * from "./components/fast-element.js";
 export * from "./components/fast-definitions.js";
 export * from "./components/attributes.js";
-export { ElementController } from "./components/element-controller.js";
+export {
+    ElementController,
+    ElementControllerStrategy,
+} from "./components/element-controller.js";
+export { addHydrationSupport } from "./components/hydration.js";
```

Comment:
I think the hydration support should not be part of the core but rather provided as an optional through export path.